com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImplTest.java

```diff
@@ -191,7 +194,12 @@ public void testExecuteOperationActionUnauthorized() {
 
     @Test
     public void testStartOperation() {
-        testedClass.startOperation(mockHttpServletRequest(EXAMPLE_USER), SPACE_GUID, createOperation(null, null));
+        Map<String, Object> parameters = new HashMap<>();
+        parameters.put(Constants.PARAM_MTA_ID, "test");
+        Operation operation = createOperation(null, null, parameters);
```

Comment:
If the parameters is only going to be a one-entry map, you could use MapUtil.asMap

Reply:
Done